packages/cli/src/lib/cmds/doctor.ts

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright 2020 Google Inc. All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+import {Log} from '@bubblewrap/core';
+import {join} from 'path';
+import {existsSync, promises as fsPromises} from 'fs';
+import {loadOrCreateConfig} from '../config';
+
+async function jdkDoctor(log: Log): Promise<boolean> {
+  const config = loadOrCreateConfig();
+  const jdkPath = (await config).jdkPath;
+  // Checks if the given path is a real path.
+  if (!existsSync(jdkPath)) {
+    log.error('The jdkPath doesn\'t exist, please run the' +
+    'following command to update it:\n' +
+    'bubblewrap updateConfig --jdkPath path/here. after thet run bubblewrap doctor again');
+    return false;
+  }
+  const file = await fsPromises.readFile(join(jdkPath, 'release'), 'utf-8');
+  if (!file) {
+    log.error('The jdkPath isn\'t correct, please run the' +
+    'following command to update it:' +
+    '\nbubblewrap updateConfig --jdkPath path/here, such that the folder "here" ' +
+    'contains the file "release.txt". after thet run bubblewrap doctor again');
+    return false;
+  };
+  // Checks if the jdk's version is 8 as needed.
+  if (file.indexOf('JAVA_VERSION="1.8') < 0) {
+    log.error('Wrong jdk version. You should download "OpenJDK' +
```

Comment:
You're a little inconsistent with where you're breaking the newlines into strings. Whichever IDE you're using, there should be a setting to enable a "vertical ruler" - set this up for 100 characters. It will put a line down the page/screen that tells you what the character limit is.

(eg, for VSCode, instructions: https://stackoverflow.com/a/29972073 )